server/src/instant/reactive/session.clj

```diff
@@ -548,7 +533,6 @@
                                :http-req http-req
                                :ws-conn ws-conn
                                :receive-q receive-q
-                               :ping-job (start-ping-job store-conn id)
                                :pending-handlers pending-handlers}]
                    (on-open store-conn socket)))
       :on-message (fn [{:keys [data]}]
```

Comment:
Should we update the `atomic-last-received-at` in `on-message` also?

Reply:
I pushed this down to the [caller of on-message](https://github.com/instantdb/instant/pull/207/files#diff-bef8276a9e702296afb121d63ec060dac0cca22a08b6e20994e347e62f810bf8R51-R52), so this already happens. Took the inspiration of updating on every message from Precursor : )